requirements.in

```diff
@@ -1 +1,2 @@
 psutil  # Gives more accurate dask.system.CPU_COUNT
+cupy-cuda12x  # Not listed in package metadata because CUDA version is unknown
```

Comment:
I'll openly admit, 
* I am still getting used to the idea that [cupy-cuda12x](https://pypi.org/project/cupy-cuda12x/) itself has a version, and
* Not to be confused with the version of CUDA it supports (also in its name).